src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykAuthPanel.kt

```diff
@@ -93,8 +94,31 @@ class SnykAuthPanel(project: Project) : JPanel(), Disposable {
                     authButton.isEnabled = true
                 }
             })
+    }
+
+    private fun authenticateEvent(): AuthenticateButtonIsClicked {
+        return builder().ide(Ide.JETBRAINS).eventSource(EventSource.IDE).build()
+    }
 
-        add(authButton, baseGridConstraints(3))
+    private fun descriptionLabelText(): String {
+        if (amplitudeExperimentService.isPartOfExperimentalWelcomeWorkflow()) {
+            return """
+        |<html><ol>
+        |  <li align="left">Authenticate to Snyk.io</li>
+        |  <li align="left">Analyze code for issues and vulnerabilities</li>
+        |  <li align="left">Improve you code and upgrade dependencies</li>
+        |</ol>
+        |</html>
+        """.trimMargin()
```

Comment:
alignment with return? also trimIndetnt() seems to work better here

Reply:
IntelliJ added that automatically when separating the lines for """. Honestly, it worked, and I don't even know the difference between the two :).

Reply:
trimIndent() doesn't need `|` symbol
also Idea doesn't know about alignment while dealing with raw string, so manual alignment needed :(